Content.Server/Motd/MOTDCommand.cs

```diff
@@ -0,0 +1,28 @@
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+
+namespace Content.Server.Motd;
+
+/// <summary>
+/// A console command which acts as an alias for <see cref="GetMotdCommand"/> or <see cref="SetMotdCommand"/> depending on the number of arguments given.
+/// </summary>
+[AnyCommand]
+internal sealed class MOTDCommand : LocalizedCommands
+{
+    public override string Command => "motd";
+    
+    public override void Execute(IConsoleShell shell, string argStr, string[] args)
```

Comment:
Does this really need to exist? It's going to show up in the command prompt even when people don't have perms for set-motd.

Reply:
Do you mean the autocomplete or the list command? The last time I played I typed list into the console and _all_ off the commands showed up... I am not an admin for reference.

Reply:
Switched it to only display the prompt for set-motd if the player can run set-motd.